src/db/src/schema.rs

```diff
@@ -16,7 +16,7 @@ pub struct Run {
     pub problem_id: ProblemId,
     pub rejudge_id: i32,
     pub user_id: UserId,
-    pub contest_id: ContestId,
+    pub contest_name: ContestName,
```

Comment:
I think it should be left ContestId for consistency with other types (Contest still has ID: name **is** identifier)